### ISSUE TITLE
Applying method to all bodies to rigid body.

### ISSUE THREAD

**Issue by hyunsooda** [liabru/matter-js#434]
Hello, I'm universty student of Korea.

I would like to make 2D Object body in person.
For example, not based are in such as Bodies.rectangle or Bodies.circle or Bodies.fromVertices .

By the way, I wondering that could make it
For example, such as circle or rectangle or polygon etc... can make body, but not make more than complicated and beautiful 2D Object body also curve.

Therefore I would like to apply rigid body to all bodies I made.

There is a method ?

**Comment by liabru** [comment]
Take a look at the [SVG example](http://brm.io/matter-js/demo/#svg), I think that might be what you're looking for?

**Comment by hyunsooda** [comment]
<img width="765" alt="2017-06-24 1 40 26" src="https://user-images.githubusercontent.com/12508269/27493030-3ea849ec-5883-11e7-8ab8-2ee908d7b37f.png">

**Comment by hyunsooda** [comment]
uhm… SVG example is    similiar I asked, but I will again ask, because problem was not solved.

First, I would like to talk SVG example.
I have hard executed SVG example including decomp,pathseg,Iconmonstr

There is very small that svg Iconmonstr is provided.
but I found svg of big size in your Git.

I have continue tried to display a small svg is to big svg.
At result, I solved it through below module. (https://www.npmjs.com/package/scale-svg-path)

Although image is display, it is slightly broken.
<img width="765" alt="2017-06-24 1 40 26" src="https://user-images.githubusercontent.com/12508269/27493040-4a30138a-5883-11e7-941b-fd2f371c2ed5.png">

How you did in SVG example based in Iconmonstr?
